src/chapters/databases-part2/home-page.rst

```diff
@@ -0,0 +1,107 @@
+Create the Home Page
+====================
+
+The Minesweeper game uses two different webpages: a home screen plus an active
+playing space. We need to build an HTML template for each page.
+
+Review ``base.html``
+--------------------
+
+Since both game pages use the same ``<head>`` element, we can put that common
+code into a base template:
+
+.. admonition:: Example
+
+   Open ``base.html`` file to find this code:
+
+   .. sourcecode:: HTML
+      :linenos:
+
+      <!DOCTYPE html>
+      <html lang="en">
+      <head>
+         <head>
+            <meta charset="UTF-8">
+            <meta name="viewport" content="width=device-width">
+            <title>LCHS Minesweeper</title>
+            <link rel="stylesheet" type="text/css"
+               href="{{ url_for('static', filename='style.css') }}">
+         </head>
+      </head>
+      <body>
+         <h1>{{page_title}}</h1>
+         {% block content %}
+         {% endblock %}
+      </body>
+      </html>
+
+Note the following:
+
+#. We link to a ``style.css`` file in line 8. It contains some basic styling
+   rules for the game, and the tutorials won't instruct you to change any of
+   the settings. However, you absolutely CAN adjust the properties and values
+   as you see fit! Don't like the gray background on the board? Fix it!
+#. On line 13, the placeholder for the page title needs to be filled. Don't
+   forget to pass a value to ``page_title`` from each ``render_template()``
+   function.
```

Comment:
A link to instructions for review would be nice

Reply:
Done!